client/src/deploy/sign.rs

```diff
@@ -0,0 +1,36 @@
+use clap::{App, ArgMatches, SubCommand};
+
+use super::creation_common;
+use crate::{command::ClientCommand, common};
+
+enum DisplayOrder {
+    SecretKey,
+    Input,
+    Output,
+}
```

Comment:
Likewise, we can probably delete this enum and use `creation_common::DisplayOrder::SecretKey`, with the other two variants not required.